components/TopNav.vue

```diff
@@ -1,40 +1,42 @@
 <template>
-    <div class="p-4 border-b flex justify-between">
+    <div class="p-4 border-b flex justify-between items-center">
         <div class="font-semibold text-xl text-gray-800">
             <NuxtLink class="align-middle" to="/">
-                <img class="inline align-middle pr-2"
-                    src="../assets/images/logo-32-px-color.svg" />Find a Doc
+                <img
+                    class="inline align-middle pr-2"
+                    src="../assets/images/logo-32-px-color.svg"
+                />Find a Doc
             </NuxtLink>
         </div>
-        <div>
-            <NuxtLink to="/about">
-                About
-            </NuxtLink>
-            <div class="cursor-pointer md:hidden">
-                <img src="../assets/images/hamburgerMenu.svg" alt="hamburger menu" />
+        <div class="flex gap-4 mx-6 absolute right-20">
+            <NuxtLink to="/about" class="px-2 mx-auto"> About Us</NuxtLink>
+
+            <div class="hidden cursor-pointer md:hidden">
+                <img
+                    src="../assets/images/hamburgerMenu.svg"
+                    alt="hamburger menu"
+                />
             </div>
         </div>
         <LocaleSelector />
     </div>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue'
+import { defineComponent } from "vue";
```

Comment:
To jump in on LaShawn's comment, we don't want to introduce style changes between different code authors, so we use a tool called eslint for consistency. Notice you accidentally changed the quotes here, and that distracts from the meaningful code changes. Please run both `yarn prepare` and `yarn lint --fix`

Thanks 🙏

Reply:
Yarn prepare is a command you only need to run once, to set up githooks that check the code on push. Yarn lint is a command you'll run often